vividus-tests/src/main/resources/story/integration/TableTransformers.story

```diff
@@ -263,6 +263,11 @@ Then `${cartesianProductTable}` is equal to table:
 |3 |AWS    |Neptune |row21|row22|row23|
 |3 |AWS    |Neptune |row31|row32|row33|
 
+Scenario: Verify CARTESIAN_PRODUCT transformer with empty table (should be skipped)
```

Comment:
```suggestion
Scenario: Verify CARTESIAN_PRODUCT transformer with empty table (should not be executed)
```

to avoid collision with a special state `skipped` for steps/scenarios